streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -237,25 +236,6 @@ public void shouldNotRemoveStatisticsFromInjectedMetricsRecorderOnCloseWhenUserP
         verify(metricsRecorder);
     }
 
-    @Test
-    public void shouldRespectBulkloadOptionsDuringInit() {
-        rocksDBStore.init(context, rocksDBStore);
-
-        final StateRestoreListener restoreListener = context.getRestoreListener(rocksDBStore.name());
-
-        restoreListener.onRestoreStart(null, rocksDBStore.name(), 0L, 0L);
-
-        assertThat(rocksDBStore.getOptions().level0FileNumCompactionTrigger(), equalTo(1 << 30));
```

Comment:
Were the bulk loading options a public interface? It doesn't seem like ignoring them would be a "breakage", so I don't think it hinders the removal, but it does seem worth documenting, and maybe trying to detect and warn?

Reply:
Sounds good. I was thrown off by `userSpecifiedOptions.prepareForBulkLoad()` because it looks like the user could have specified some bulk loading options, but I took another look, and I see that that's just the variable name. The actual method is internal, and there's no way to configure bulk-loading options specifically as a user. So, I'm satisfied. Thanks!